packages/sdks/src/blocks/personalization-container/helpers.ts

```diff
@@ -91,30 +126,46 @@ export function getBlocksToRender({
   // If we're on the browser, check if there's a winning variant
   if (isBrowser()) {
     const winningVariant = filteredVariants?.[0];
-    if (winningVariant) {
-      return {
-        blocks: winningVariant.blocks,
-        path: `component.options.variants.${variants?.indexOf(winningVariant)}.blocks`,
-      };
+    if (winningVariant && variants) {
+      const variantIndex = variants.indexOf(winningVariant);
+      if (variantIndex !== -1) {
+        return {
+          blocks: winningVariant.blocks,
+          path: `component.options.variants.${variantIndex}.blocks`,
```

Comment:
we prefix `component.options` in Blocks, so you can remove it from all the paths in this file

Reply:
done!